src/js/helpers/oboeWrapper.js

```diff
@@ -1,28 +1,53 @@
 var oboe = require("oboe");
 
 var oboeWrapper = function (req) {
+  var response = {
+    status: null,
+    body: null
+  };
+
+  var parseResponse = function (xhr) {
+    response.status = xhr.statusCode;
+    try {
+      response.body = JSON.parse(xhr.body);
+    } catch (e) {
+      response.body = xhr.body;
+    }
+  };
+
   var api = oboe(req)
     .start(function (status) {
       this.status = status;
     });
 
   api.success = function (callback) {
-    this.done(function (payload) {
+    this.done(function (json) {
       if (!this.status.toString().match(/^2[0-9][0-9]$/)) {
```

Comment:
But actually I want to rewrite the regexp to 
` if (!this.status.toString().match(/^[^2/)) {`